packages/flutter/test/material/bottom_navigation_bar_theme_test.dart

```diff
@@ -102,7 +102,7 @@ void main() {
     expect(description[8], 'showSelectedLabels: true');
```

Comment:
For the tests in the `material` directory, should they use MaterialState instead of WidgetState since the material one should be a subclass of the widget one?